src/Exception/ErrorCodeExceptionInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Exception;
+
+/**
+ * Interface ErrorCodeExceptionInterface.
+ */
+interface ErrorCodeExceptionInterface extends \Throwable
```

Comment:
```suggestion
interface ErrorCodeAvailableInterface
```

Not sure about this name but it should not extend `\Throwable`. This interface should be an optional interface which custom exception classes choose to implement in addition to the usual.

Reply:
updated the name and removed the extends.